README.md

```diff
@@ -1,4 +1,7 @@
 # Amazon QLDB Java DMV Sample App
+[![license](https://img.shields.io/badge/license-MIT-green)](https://github.com/aws-samples/amazon-qldb-dmv-sample-java/blob/master/LICENSE)
+[![license](https://img.shields.io/badge/license-Apache%202.0-blue)](https://github.com/awslabs/amazon-qldb-driver-java/blob/master/LICENSE)
```

Comment:
this is referencing to the driver's license file.

Reply:
I think this should use the MIT/Apache shield instead of two shields: https://img.shields.io/badge/license-MIT%2FApache--2.0-blue